airflow/configuration.py

```diff
@@ -344,6 +353,9 @@ def getsection(self, section):
             _section[key] = val
         return _section
 
+    def _handle_opt(self):
+        pass
```

Comment:
⁉️

Reply:
Sorry about that, I can't remember what I was thinking at the time. This weekend I will remove it